RadixWallet/Prelude/Extensions/Collection+Extra.swift

```diff
@@ -33,6 +33,17 @@ extension OrderedSet where Element: Hashable {
 	}
 }
 
+extension IdentifiedArray {
+	/// Add or remove the given element
+	public mutating func toggle(_ element: Element) {
```

Comment:
I'd prefer another name than `toggle`, I did not even react when I read up above, thought it was a bool.... hmm...

I think `togglePresence:of` is better (not great... but better)